app/views/user_message_statuses/index.html.haml

```diff
@@ -13,4 +13,4 @@
         %td
           = user_message_status.message.created_at.strftime(Settings.date_time_format)
         %td
-          = link_to 'View claim', __send__("#{current_user.persona.class.to_s.pluralize.underscore}_claim_path", user_message_status.message.claim) + "#messages"
+          = link_to 'View claim', __send__(current_user.persona.message_claim_path, user_message_status.message.claim, anchor: 'messages')
```

Comment:
I'd be tempted to put a message_claim_path on the base class that does this dynamically (albeit that class names are not likely to change (though these they did once).
```
def message_claim_path
  "#{class.to_s.pluralize.underscore}_claim_path"
end
```

...unless brakeman no-likey